pkg/node/node.go

```diff
@@ -474,34 +595,132 @@ func (n *Node) monitorKeepPublicKeySubmission(
 	}
 
 	defer subscriptionConflictingPublicKey.Unsubscribe()
-	defer subscriptionPublicKeyPublished.Unsubscribe()
 
-	select {
-	case event := <-publicKeyPublished:
-		logger.Infof(
-			"public key [%x] has been accepted by keep: [%s]",
-			event.PublicKey,
-			keepAddress.String(),
-		)
-	case event := <-conflictingPublicKey:
-		logger.Errorf(
-			"member [%x] has submitted conflicting public key for keep [%s]: [%x]",
-			event.SubmittingMember,
-			keepAddress.String(),
-			event.ConflictingPublicKey,
-		)
-	case <-monitoringCtx.Done():
-		logger.Warningf(
-			"monitoring of public key submission for keep [%s] "+
-				"has been cancelled: [%v]",
-			keepAddress.String(),
-			monitoringCtx.Err(),
-		)
-	case <-abort:
-		logger.Warningf(
-			"monitoring of public key submission for keep [%s] "+
-				"has been aborted",
-			keepAddress.String(),
-		)
+	pubkeyChecksCounter := 0
+	const maxPubkeyChecksCount = 3
+	const pubkeyCheckTick = 10 * time.Minute
```

Comment:
I'd note why we are doing it less aggressive than for signature submission:
```
	// All three operators need to submit public key to the chain so we are
	// less aggressive with check ticks than in case of signature submission
	// where only one signature is enough.
```